pymc_marketing/clv/models/beta_geo_beta_binom.py

```diff
@@ -0,0 +1,677 @@
+                "gamma_prior": Prior("HalfFlat"),
+                "delta_prior": Prior("HalfFlat"),
+            },
+        )
+
+        # Fit model quickly to large datasets via the default Maximum a Posteriori method
+        model.fit(fit_method='map')
+        print(model.fit_summary())
+
+        # Use 'mcmc' for more informative predictions and reliable performance on smaller datasets
+        model.fit(fit_method='mcmc')
+        print(model.fit_summary())
+
+        # Predict number of purchases for customers over the next 10 time periods
+        expected_purchases = model.expected_purchases(
+            data=rfm_df,
+            future_t=10,
+        )
+
+        # Predict probability of customer making 'n' purchases over 't' time periods
+        # Data parameter is omitted here because predictions are ran on original dataset
+        expected_num_purchases = model.expected_purchase_probability(
+            n=[0, 1, 2, 3],
+            future_t=[10,20,30,40],
+        )
+
+        new_data = pd.DataFrame(
+            data = {
+            "customer_id": [0, 1, 2, 3],
+            "frequency": [5, 2, 1, 8],
+            "recency": [7, 4, 2.5, 11],
+            "T": [10, 8, 10, 22]
+            }
+        )
+
+        # Predict probability customers will still be active in 'future_t' time periods
+        probability_alive = model.expected_probability_alive(
+            data=new_data,
+            future_t=[0, 3, 6, 9],
+        )
+
+        # Predict number of purchases for a new customer over 't' time periods.
+        expected_purchases_new_customer = model.expected_purchases_new_customer(
+            t=[2, 5, 7, 10],
+        )
+
+    References
+    ----------
+    .. [1] Peter Fader, Bruce Hardie, and Jen Shang.
+           "Customer-Base Analysis in a Discrete-Time Noncontractual Setting".
+           Marketing Science, Vol. 29, No. 6 (Nov-Dec, 2010), pp. 1086-1108.
+           https://www.brucehardie.com/papers/020/fader_et_al_mksc_10.pdf
+    """
+
+    _model_type = "BG/BB"  # Beta-Geometric, Beta-Binomial Distribution
+
+    def __init__(
+        self,
+        data: pd.DataFrame,
+        *,
+        model_config: ModelConfig | None = None,
+        sampler_config: dict | None = None,
+    ):
+        super().__init__(
+            data=data,
+            model_config=model_config,
+            sampler_config=sampler_config,
+            non_distributions=None,
+        )
+        self._validate_cols(
+            data,
+            required_cols=[
+                "customer_id",
+                "frequency",
+                "recency",
+                "T",
+            ],
+            must_be_unique=["customer_id"],
+            must_be_homogenous=["T"],
+        )
+
+    @property
+    def default_model_config(self) -> ModelConfig:
+        return {
+            "alpha_prior": None,
+            "beta_prior": None,
+            "gamma_prior": None,
+            "delta_prior": None,
```

Comment:
Can you have these absent from the dictionary instead of having set to non. Else you can use the `non_distributions` in the parsing of the parent init